package.json

```diff
@@ -75,7 +75,7 @@
     "run-sequence": "2.2.1",
     "sinon": "7.4.1",
     "tslint": "5.19.0",
-    "typescript": "^3.5.3",
+    "typescript": "3.0.3",
```

Comment:
Why the version drop ?

Reply:
https://github.com/inversify/InversifyJS/pull/1130

Reply:
thanks